File: app/src/main/java/com/beekay/thoughts/model/Reminder.java
package com.beekay.thoughts.model;

public class Reminder {

    private Integer id;
    private String reminderText;
    private boolean status;
    private String toBeDoneOn;
    private String createdOn;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getReminderText() {
        return reminderText;
    }

    public void setReminderText(String reminderText) {
        this.reminderText = reminderText;
    }

    public boolean isStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public String getToBeDoneOn() {
        return toBeDoneOn;
    }

    public void setToBeDoneOn(String toBeDoneOn) {
        this.toBeDoneOn = toBeDoneOn;
    }

    public String getCreatedOn() {
        return createdOn;
    }

    public void setCreatedOn(String createdOn) {
        this.createdOn = createdOn;
    }
}

File: app/src/main/java/com/beekay/thoughts/model/Reminder.kt
package com.beekay.thoughts.model

import androidx.annotation.NonNull
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Created by Krishna by 15-11-2020
 */
@Entity(tableName = "reminder")
data class Reminder(
        @PrimaryKey(autoGenerate = true)
        @ColumnInfo(name = "id")
        @NonNull
        val id: Long = 0,

        @ColumnInfo(name = "reminder_about")
        @NonNull
        val reminderAbout: String,

        @ColumnInfo(name = "date_when")
        @NonNull
        val toBeDoneOn: String,

        @ColumnInfo(name = "createdon")
        @NonNull
        val createdOn: String,

        @ColumnInfo(name = "tag")
        @NonNull
        val tag: String,

        @ColumnInfo(name = "status")
        @NonNull
        val status: Boolean
) {
}